Scenarios/Authentication/Service-to-service-JWT-using-Azure-AD/TileService/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug" 
+        /*,
+      "Default": "Information",
```

Comment:
this is not valid json

Reply:
Config supports comments, so its not strict json, but works.